.clang-format

```diff
@@ -94,10 +94,6 @@ PenaltyBreakString: 1000
 PenaltyExcessCharacter: 1000000
 PenaltyReturnTypeOnItsOwnLine: 200
 PointerAlignment: Left
-RawStringFormats:
```

Comment:
?

Reply:
They changed something in clang 9 and I hoped to sneak it in.